content/docs/guides/liquibase.md

```diff
@@ -0,0 +1,294 @@
+        first_name VARCHAR(100),
+        last_name VARCHAR(100),
+        email VARCHAR(255) UNIQUE NOT NULL,
+        bio TEXT
+    );
+
+    -- Creating the `posts` table
+    CREATE TABLE posts (
+        post_id SERIAL PRIMARY KEY,
+        author_id INTEGER REFERENCES authors(author_id),
+        title VARCHAR(255) NOT NULL,
+        content TEXT,
+        published_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+    );
+    ```
+
+## Retrieve your Neon database connection string
+
+From the Neon **Dashboard**, retrieve your password and a Java connection string from the **Connection Details** widget. Use the selection drop-down menu.
+
+Your Java connection string should look something like the one shown below.
+
+<CodeBlock shouldWrap>
+
+```bash
+jdbc:postgresql://ep-cool-darkness-123456.us-east-2.aws.neon.tech/blog?user=alex&password=AbC123dEf
+```
+
+</CodeBlock>
+
+## Connect from Liquibase to your Neon database
+
+1. Create a directory for your Liquibase project. For example:
+
+    ```bash
+    mkdir blogdb
+    ```
+
+2. Change to your project directory and create a `liquibase.properties` file.
+
+    ```bash
+    cd blogdb
+    touch liquibase.properties
+    ```
+
+3. Open the the `liquibase.properties` file in an editor and add entries for a [liquibase changelog file](https://docs.liquibase.com/concepts/changelogs/home.html) and your database `url`. We'll call the changelog file `dbchangelog.xml`. You will use this file to define schema changes. For the `url`, specify the Neon connection string you retrieved previously.
+
+    <CodeBlock shouldWrap>
+
+    ```env
+    changeLogFile:dbchangelog.xml  
+    url: jdbc:postgresql://ep-floral-poetry-66238369.us-east-2.aws.neon.tech/blog?user=alex&password=4GfNAqycba8P&sslmode=require
+    ```
+
+    </CodeBlock>
+
+## Take a snapshot of your database
+
+In this step, you will run the [generateChangelog](https://docs.liquibase.com/commands/inspection/generate-changelog.html) command in your project directory to create a changelog file with the current state of your database. We'll call this file `mydatabase_changelog.xml`.
+
+```bash
+liquibase --changeLogFile=mydatabase_changelog.xml generateChangeLog
+```
+
+You’ll get a changelog file for your database that looks something like this:
+
+```xml
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-latest.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="alex (generated)" id="1697969580160-1">
```

Comment:
Add a note (or highlight in the sample) a reminder to use your own details for author + id (you can grab them from the generated XML file in the previous step).

Reply:
Added a reminder